src/Sulu/Bundle/MediaBundle/Resources/js/views/MediaDetail/MediaDetail.js

```diff
@@ -0,0 +1,107 @@
+// @flow
+import React from 'react';
+import {observer} from 'mobx-react';
+import {translate} from 'sulu-admin-bundle/services';
+import {withToolbar} from 'sulu-admin-bundle/containers';
+import type {ViewProps} from 'sulu-admin-bundle/containers';
+import {ResourceStore} from 'sulu-admin-bundle/stores';
+import MediaUploadStore from '../../stores/MediaUploadStore';
+import SingleMediaDropzone from '../../components/SingleMediaDropzone';
+import mediaDetailStyles from './mediaDetail.scss';
+
+type Props = ViewProps & {
+    resourceStore: ResourceStore,
+};
+
+@observer
+class MediaDetail extends React.PureComponent<Props> {
+    mediaUploadStore: MediaUploadStore;
+
+    componentWillMount() {
+        const {
+            router,
+            resourceStore,
+        } = this.props;
+
+        router.bind('locale', resourceStore.locale);
+        this.mediaUploadStore = new MediaUploadStore(resourceStore);
+    }
+
+    handleMediaDrop = (file: File) => {
+        const {resourceStore} = this.props;
+        const {
+            data: {
+                id: mediaId,
+            },
+        } = resourceStore;
+
+        this.mediaUploadStore.update(mediaId, file);
+    };
+
+    render() {
+        const {
+            source,
+            progress,
+            uploading,
+        } = this.mediaUploadStore;
+
+        return (
+            <div className={mediaDetailStyles.mediaDetail}>
+                <section>
+                    <SingleMediaDropzone
+                        source={source}
+                        uploading={uploading}
+                        progress={progress}
+                        onDrop={this.handleMediaDrop}
+                        uploadText={translate('sulu_media.upload_or_replace')}
+                    />
+                </section>
+            </div>
+        );
+    }
+}
+
+export default withToolbar(MediaDetail, function() {
+    const {
+        router,
+        resourceStore,
+    } = this.props;
+    const {backRoute, locales} = router.route.options;
+
+    const backButton = backRoute
+        ? {
+            onClick: () => {
+                router.restore(backRoute, {locale: resourceStore.locale.get()});
```

Comment:
Instead of having the `backRoute` being configureable here, that should also be hardcoded (that's also the reason there is currently no back button in this view).